contracts/GenArt721CoreV3.sol

```diff
@@ -407,6 +411,29 @@ contract GenArt721CoreV3 is
         ownerAndHashSeed.hashSeed = bytes12(_hashSeed);
     }
 
+    /**
+     * @notice Allows owner (AdminACL) to revoke ownership of the contract.
+     * Note that the contract is intended to continue to function after the
+     * owner renounces ownership, but no new projects will be able to be added.
+     * Renouncing ownership will leave the contract without an owner,
+     * thereby removing any functionality that is only available to the
+     * owner/AdminACL contract. The same is true for any dependent contracts
+     * that also integrate with the owner/AdminACL contract (e.g. potentially
+     * minter suite contracts, registry contracts, etc.).
+     * After renouncing ownership, artists will be in control of updates to
+     * their payment addresses and splits (see modifier
+     * onlyAdminACLOrRenouncedArtist`)
```

Comment:
I think that this description is great as-is. I wonder if we should even more explicitly add something like:

"While there is no currently intended reason to call this method based on defined Art Blocks business practices, this method exists to allow artists to continue to maintain the limited set of contract functionality that exists post-project-lock in an environment in which there is no longer an admin maintaining this smart contract." 

WDYT, this too verbose/not helpful?

Reply:
I like it! added and pushed, will merge with these updates.